docs/source/cross_validation.rst

```diff
@@ -24,6 +24,47 @@ The interface for splitting Dask arrays is the same as scikit-learn's version.
 
    X_train.compute()[:3]
 
+Here is another illustration of performing k-fold cross validation purely in Dask. Here a link to gather more information on k-fold cross validation :func:`https://ml.dask.org/modules/generated/dask_ml.model_selection.KFold.html`:
+
+.. ipython:: python
+
+   import dask.array as da
+   from dask_ml.model_selection import KFold
+   from dask_ml.datasets import make_regression
+   from dask_ml.linear_model import LinearRegression
+   from statistics import mean 
+
+   X, y = make_regression(n_samples=200, # choosing number of observations
+				 n_features=5, # number of features
+				 random_state=0, # random seed
+				 chunks=20) # partitions to be made 
+
+   train_scores: list[int] = []
+   test_scores: list[int] = []
+
+   model = LinearRegression()
+
+The Dask kFold method splits the data into k consecutive subsets of data. Here we specify k to be 5, hence, 5-fold cross validation
+
+.. ipython:: python
+   kf = KFold(n_splits=5)
+
+   for i, j in kf.split(X):
+      X_train, X_test = X[i], X[j]
```

Comment:
Can you check that the formatting here is correct? (might need another space)

Reply:
you are correct, it needed another space. updated the code to reflect the same.